tests/docker/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# PHP Version number as set by offical php docker image tags: https://hub.docker.com/r/library/php/tags/
+# Apache is setup as http server, varnish + nginx is installed for use as reverse proxies
+ARG PHP_VERSION=7.1
+FROM php:${PHP_VERSION}-apache
```

Comment:
This requires Docker >= 17.05.0, so you need to install that (Travis now runs 17.03).

Reply:
Travis is running on 17.09 since the 12th ;)

Reply:
nice when problems solve themselves :-)